src/assets/css/global.css

```diff
@@ -151,12 +151,14 @@ textarea:focus {
   display: flex;
   align-items: center;
   justify-content: center;
+  flex-wrap: wrap;
 }
 
 .flex-items a {
   display: block;
-  margin: 0 20px;
+  margin: 20px;
```

Comment:
We could also consider using rem here due its calculated on the normal-font size

Reply:
Okay. Will make the change